src/nvim/normal.c

```diff
@@ -399,7 +399,8 @@ void init_normal_cmds(void)
 
 void init_default_mappings(void)
 {
-  add_map((char_u *)"Y y$", NORMAL);
+  add_map((char_u *)"Y y$", NORMAL, true);
+  add_map((char_u *)"<C-L> <Cmd>nohlsearch<Bar>diffupdate<CR><C-L>", NORMAL, true);
```

Comment:
- `<cmd>` is a good idea
- do we need `<silent>` (extra arg to add_map I guess? but maybe it's not needed)
- what about the ternary logic in https://github.com/tpope/vim-sensible/blob/2d9f34c09f548ed4df213389caa2882bfe56db58/plugin/sensible.vim#L35

Reply:
> * do we need <silent> (extra arg to add_map I guess? but maybe it's not needed)

For this particular mapping, I don't think so. Testing locally on my end at least does not print any output.

> * what about has('diff') to avoid unnecessary :diffupdate?

AFAIK `has('diff')` should always be true in Nvim, so this isn't necessary.

Reply:
> `has('diff')` checks for the capability to do diffs

🤦  I was thinking about `&diff`, never mind!